CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## [0.12.0] - 2023-12-19
 
+### Fixed
+
+-   :bug: Add support for dataset factories in KedroPipelineML ([#516, rxm7706](https://github.com/Galileo-Galilei/kedro-mlflow/pull/519))
```

Comment:
I think you should put your name instead of rxm7706 if you want to be credited ;)

Reply:
ahh thanks! I also moved it to the Unreleased section